### ISSUE TITLE
The `refresh dbt` button does not appear on the homepage

### ISSUE THREAD

**Issue by TuringLovesDeathMetal** [lightdash/lightdash#1376]
### Describe the bug

In the docs, when you're setting up Lightdash, we tell users to `refresh dbt` once they've added their first metrics + dimensions. But, when you go to the Lightdash homepage, the `refresh dbt` button isn't there - you have to go to `Explore` to see it. 

### To Reproduce
This is what a user reads in the docs:
<img width="823" alt="image" src="https://user-images.githubusercontent.com/31848148/154029062-93f87c6c-9136-48ad-b6a7-cbb2d073ba5a.png">

Then this is what they see on the homepage: 
<img width="679" alt="image" src="https://user-images.githubusercontent.com/31848148/154029165-336da49d-fd9c-46d8-93ae-a9709fc80c39.png">

### Expected behavior

There should be a `refresh dbt` button on the Lightdash homepage.

**Comment by rephus** [comment]
I think I can start looking into this

**Comment by rephus** [comment]
Right, so correct me if I'm wrong, but has this page changed ? The onboarding I see now looks like this 
![Screenshot 2022-03-22 at 09 50 22](https://user-images.githubusercontent.com/1983672/159447821-2a1020ee-e2f4-4a63-8ce1-339111e75cf6.png)
I also tried in non-dev mode

**Comment by ZeRego** [comment]
@rephus correct, the onboarding page changed since this ticket was created. @TuringLovesDeathMetal @PriPatel is this ticket still relevant ?

**Comment by PriPatel** [comment]
@ZeRego @TuringLovesDeathMetal @rephus 

Thanks. I think its still relevant but with a couple of tweaks! 
As we've updated the 'onboarding' screen to have 1 step (and therefore be easier to dismiss), it makes sense that the `Refresh dbt` button appears on the homepage. It can go next to the `run a query` button. 


<img width="745" alt="Screenshot 2022-03-22 at 12 01 17" src="https://user-images.githubusercontent.com/12776522/159477414-c272e027-5206-4664-b43f-ed05d3c1eba8.png">

**Comment by rephus** [comment]
@PriPatel do you have any updated design for this ?

**Comment by PriPatel** [comment]
@rephus - I dont actually, sorry! Especially seeing now as the button has changed completely!

**Comment by PriPatel** [comment]
We could close this as I'll be speccing some navbar changes soon? 

<img width="354" alt="Screenshot 2022-08-08 at 16 38 39" src="https://user-images.githubusercontent.com/12776522/183456908-de49ee0c-ee0e-4213-9e52-8988b7b1c2e1.png">
